edgelet/edgelet-docker/tests/runtime.rs

```diff
@@ -775,6 +775,13 @@ fn container_create_handler(req: Request<Body>) -> ResponseFuture {
                         .unwrap()
                 );
 
+                assert!(create_options
+                    .networking_config()
+                    .unwrap()
+                    .endpoints_config()
+                    .unwrap()
+                    .contains_key(&"test-network-id".to_string()));
```

Comment:
`&"test-network-id".to_string()` -> `"test-network-id"`, because `HashMap::contains_key` allows you to use a `Borrow`, ie `str`

Reply:
Excellent, will do.